Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM golang:1.19.2-bullseye AS build-env
+
+# Install minimum necessary dependencies,
+ENV PACKAGES make git gcc
+RUN apt-get update -y
+RUN apt-get install -y $PACKAGES
+
+COPY . /src/panacea-dep-consumer
+
+WORKDIR /src/panacea-dep-consumer
+
+RUN make clean && make build
+
+FROM debian:bullseye-slim
+
+COPY --from=build-env /src/panacea-dep-consumer/build/consumerd /usr/bin/consumerd
+
+RUN chmod +x /usr/bin/consumerd
+
+CMD ["/consumerd"]
+
+EXPOSE 8080
```

Comment:
Can you need this expose option?
I know that the default port for this service is 8080, which is not use.

Reply:
Yes, the user can set listen address address with port. This option is not necessary and without this option the works well. Fixed it in [970f962](https://github.com/medibloc/panacea-dep-consumer/pull/5/commits/970f9627a64c2a9947b3ba2c3812859f6b0a4cff).